[2025-10-31T02:55:20.254+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-31T02:55:20.285+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:35:10.758793+00:00 [queued]>
[2025-10-31T02:55:20.295+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:35:10.758793+00:00 [queued]>
[2025-10-31T02:55:20.296+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2025-10-31T02:55:20.313+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-10-31 02:35:10.758793+00:00
[2025-10-31T02:55:20.320+0000] {standard_task_runner.py:63} INFO - Started process 240 to run task
[2025-10-31T02:55:20.323+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgres_dag', 'load_to_postgres', 'manual__2025-10-31T02:35:10.758793+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/api_to_postgres_dag.py', '--cfg-path', '/tmp/tmp_4j3zov8']
[2025-10-31T02:55:20.326+0000] {standard_task_runner.py:91} INFO - Job 85: Subtask load_to_postgres
[2025-10-31T02:55:20.389+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_postgres_dag.load_to_postgres manual__2025-10-31T02:35:10.758793+00:00 [running]> on host 754154eb0275
[2025-10-31T02:55:20.486+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lenny' AIRFLOW_CTX_DAG_ID='api_to_postgres_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T02:35:10.758793+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T02:35:10.758793+00:00'
[2025-10-31T02:55:20.488+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-31T02:55:20.518+0000] {logging_mixin.py:188} INFO - âœ… Loaded data into PostgreSQL table.
[2025-10-31T02:55:20.520+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-31T02:55:20.521+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-31T02:55:20.533+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_postgres_dag, task_id=load_to_postgres, run_id=manual__2025-10-31T02:35:10.758793+00:00, execution_date=20251031T023510, start_date=20251031T025520, end_date=20251031T025520
[2025-10-31T02:55:20.577+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-31T02:55:20.596+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-31T02:55:20.599+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
